packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -663,6 +671,86 @@ export default class GoogleCalendarService implements Calendar {
     }
   }
 
+  async getMeetParticipants(videoCallUrl: string | null): Promise<ParticipantWithEmail[][]> {
+    const { token } = await this.oAuthManagerInstance.getTokenObjectOrFetch();
+    if (!token) {
+      throw new Error("Invalid grant for Google Calendar app");
+    }
+
+    const googleAuth = new GoogleAuth({
+      authClient: new OAuth2Client({
+        credentials: {
+          access_token: token.access_token,
+        },
+      }),
+    });
+
```

Comment:
It was throwing type error when i used the Google auth client in this class so i had to create this one